### ISSUE TITLE
version 1.1.38 settings

### ISSUE THREAD

**Issue by chathurag** [Cacti/cacti#1598]
empty settings for users in 1.1.38 or 1.2.0

![cacti](https://user-images.githubusercontent.com/35136010/39516271-15978b98-4e1a-11e8-9fba-e5a511be66f2.png)

**Comment by netniV** [comment]
This looks like you are using the classic theme?  Can you do the following queries?
```sql
select * from settings
select * from settings_user where user = '<username>'
```
replace <username> with the user in in question.

**Comment by anarkia1976** [comment]
I'm using version 1.1.38 and classic theme without problems, it seems installation problem on your box.

**Comment by chathurag** [comment]
select * from settings  gives huge list
select * from settings_user where user = '<username>' gives ERROR 1054 (42S22): Unknown column 'user' in 'where clause'


My installation is ok and I did it with git pull origin from 1.1.37 update

**Comment by anarkia1976** [comment]
Your query isn't clear.
You can try to doint a "git reset to 1.1.37 point" than "git rh and git status" than "git remote update" and then "git pull".
After these operations you have to fix files owners and permissions.
at the end "service httpd restart".
thanks

**Comment by netniV** [comment]
Yeah, the settings list will be full.  The `user = ''` should actually be `username = '<username>'` (as in the user whose having the display issue).

**Comment by chathurag** [comment]
@netniV  I used username as admin it gave me
MariaDB [cactidb]> select * from settings_user where user = 'admin';
ERROR 1054 (42S22): Unknown column 'user' in 'where clause'

@anarkia1976 - My installation was successful when I install it

**Comment by netniV** [comment]
`username`

**Comment by chathurag** [comment]
@netniV - I can see user settings by database query. Which I want to say is in the latest update in develop branch has the above issue in classic theme. Users can't see the settings via GUI.

**Comment by netniV** [comment]
Yes, I am after your settings to replicate the issue.  You can mask any sensitive info.
